src/app/legacy/containers/LinkedData/index.jsx

```diff
@@ -104,6 +105,31 @@ const LinkedData = ({
     alternateName: lang,
   };
 
+  const { authorName, authorTopicUrl, twitterLink, authorImage, location } =
+    bylineLinkedData || {};
+
+  const hasByline = !!authorName;
```

Comment:
Can `bylineLinkedData` be used to determine if byline data is available?

Reply:
Yeah that makes sense, the only issue is that authorName and jobRole have to be present to be a valid byline, so if only authorRole or jobRole existed on their own, then checking the entire data would be a false positive I think. I guess it just depends if optimo let bylines through with just those fields on there own.

I used the basis of the previous `if (!(author && jobRole)) return null;` to get that conclusion

Reply:
Is it possible to return null early in the `bylineExtractor` function when checking for `authorName1 and `jobRole`? So if both aren't present, then return `null`? 

This would probably mean you couldn't destruct the object like this since it could be `null`, so maybe not as clean.

Reply:
How about this early return in the extractor:
```tsx
const authorName = pathOrZeroIndexModelBlocks(2, 'text', authorBlock);
const jobRole = pathOrZeroIndexModelBlocks(2, 'text', jobRoleBlock);

if (!authorName || !jobRole) {
  return null;
}
```

Then this in the article page:
```tsx
const bylineLinkedData = bylineExtractor(bylineContribBlocks);

const hasByline = !!bylineLinkedData;
...
<LinkedData
  showAuthor
  bylineLinkedData={bylineLinkedData}
  ...
/>
```

Then this in the byline component:
```tsx
const bylineValues = bylineExtractor(blocks);

if (!bylineValues) return null;

const {
    authorName,
    jobRole,
    twitterText,
    twitterLink,
    authorImage,
    location,
    authorTopicUrl,
  } = bylineValues;
```

Reply:
Yea I like the look of this 👍🏼 Makes sense having it escape early if the `authorName || jobRole` condition is met and keeps all of that business logic within the extract function.